docs/userguide/containers/dockervolumes.md

```diff
@@ -109,12 +116,13 @@ followed by `a-z0-9`, `_` (underscore), `.` (period) or `-` (hyphen).
 An absolute path starts with a `/` (forward slash).
 
 For example, you can specify either `/foo` or `foo` for a `host-dir` value.
-If you supply the `/foo` value, Docker creates a bind-mount. If you supply
-the `foo` specification, Docker creates a named volume.
+If you supply the `/foo` value, Engine creates a bind-mount. If you supply
+the `foo` specification, Engine creates a named volume.
 
-If you are using Docker Machine on Mac or Windows, your Docker daemon has only limited access to your OS X or Windows filesystem. Docker Machine tries
-to auto-share your `/Users` (OS X) or `C:\Users` (Windows) directory.  So,
-you can mount files or directories on OS X using.
+If you are using Docker Machine on Mac or Windows, your Engine daemon has only
```

Comment:
Just noted; Perhaps we should mention "boot2docker" or "toolbox" here, to exclude other setups (Windows Daemon, remote Daemon, etc)